providers/implementations/keymgmt/slh_dsa_kmgmt.c

```diff
@@ -190,6 +191,13 @@ static int slh_dsa_get_params(void *keydata, OSSL_PARAM params[])
                                                 ossl_slh_dsa_key_get_pub_len(key)))
             return 0;
```

Comment:
These `return 0` when the parameter is not available (simply NULL, as opposed to encode failure) need to just skip the parameter (like in ML-DSA, ...).

Reply:
well spotted.